index.js

```diff
@@ -1,6 +1,8 @@
-const { PublishEvent, PublishEventBatch } = require('./app/event-publisher')
+const { PublishEvent, PublishEventBatch } = require('./app/v1/event-publisher')
+const { EventPublisher } = require('./app/v2')
```

Comment:
I noticed the V2 event-publisher related files weren't in an event-publisher folder like in V1, was just wondering what the reason behind this was and if it would be better to group them?

Reply:
Good point.  Initially it felt like overkill so I didn't, but it would be cleaner if the class was packaged in it's own module with the validate and schema files.

Updated.